new/rfcXXX-lifecycle-and-release-policies.md

```diff
@@ -0,0 +1,214 @@
+
+Some Chef OSS projects will have very large sets of dependencies that are allowed to float (eg. ~> version constraint). Doing so allows other projects to consume them without having to match dependencies in lock-step, helping to prevent ecosystem failure when project upgrades before another. Because these dependencies are resolved at build-time, two builds of the same Chef OSS commit may result in slightly different packages. It is for this reason that we encourage users, when possible, to consume pre-compiled binaries through official channels (where this effect is minimized) rather than compiling them themselves.
+
+#### Build Channels
+
+Chef Software maintains three artifact channels: unstable, current, and stable.
+
+  * unstable: Builds that have completed successfully and are awaiting additional automated testing.
+  * current: Builds that have passed all automated testing.
+  * stable: Builds that have been manually promoted from current and are considered "ready for production".
+
+#### Release Candidate
+
+A "Release Candidate" is an artifact of a Chef OSS project that is available for download (eg. available in the _current_ channel) that has been identified as a candidate for a Stable Release.
+
+#### Stable Release
+
+The term "Stable Release" is used to identify a build of a Chef OSS product that is:
+
+  * consumable on its own (eg. not shipped only as a part of another software artifact)
+  * deemed suitable for use in production environments
+
+The term originated from Chef Software's process of promoting a build from the _current_ channel to the _stable_ channel. However, as not all Chef OSS is built and hosted by Chef Software, the announcement of a Stable Release may also refer to the selection of a version as the "recommended" version for use in production settings.
+
+The current Stable Release for a given software product will be reflected at the top README of the Github Repository using a badge (eg. [![Gem Version](https://badge.fury.io/rb/chef.svg)](https://badge.fury.io/rb/chef))
+
+#### Hot Fix Releases
+
+Chef OSS maintainers will provide support for critical regressions and [Common Vulnerabilities and Exposures](https://cve.mitre.org/) (CVEs) for the latest `MAJOR.MINOR` and `MAJOR-1.MINOR` Stable Releases.
+
+In the case a Hot Fix Release is required, a release will be fixed following the versioning cadence for a new Stable Release as outlined by the appropriate Lifecycle Policy. Should a fix for the regression be unidentified after an initial investigation period of roughly five hours, the affected versions will be revoked and be made unavailable for download.
```

Comment:
My intent here was to direct people to the policy sections for those details since in some cases it might happen, but in other cases it won't happen.